src/librustc/dep_graph/graph.rs

```diff
@@ -492,32 +490,29 @@ impl DepGraph {
 
     pub fn edge_deduplication_data(&self) -> Option<(u64, u64)> {
         if cfg!(debug_assertions) {
-            let current_dep_graph = self.data.as_ref().unwrap().current.borrow();
+            let current_dep_graph = &self.data.as_ref().unwrap().current;
 
-            Some((current_dep_graph.total_read_count,
-                  current_dep_graph.total_duplicate_read_count))
+            Some((current_dep_graph.total_read_count.load(Acquire),
+                  current_dep_graph.total_duplicate_read_count.load(Acquire)))
```

Comment:
It'd be good to get these to be SeqCst instead of Acquire for simplicity/consistency.